packages/react-devtools-shared/src/backend/fiber/renderer.js

```diff
@@ -3463,8 +3617,10 @@ export function attach(
                     traceNearestHostComponentUpdate,
                     virtualLevel,
                   );
+                  updateFlags |=
+                    ShouldResetChildren | ShouldResetSuspenseChildren;
```

Comment:
`mountVirtualInstanceRecursively` can mount a Fiber. Was the flag previously not set on purpose or missing?

Reply:
For `previousVirtualInstanceWasMount` to be true, we must have already set this below on 3941.

You do set `ShouldResetSuspenseChildren` here but you don't on 3941. It's not necessary to set in both places but they should be consistent.

You could maybe check if there was a SuspenseNode added to the `previouslyReconciledSiblingSuspenseNode` by checking if it changed before and after `mountVirtualInstanceRecursively` to know whether you need to set `ShouldResetSuspenseChildren`.